dask_sql/physical/rel/base.py

```diff
@@ -30,7 +30,7 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> dd.DataFra
 
     @staticmethod
     def fix_column_to_row_type(
-        cc: ColumnContainer, row_type: "RelDataType"
+        cc: ColumnContainer, row_type: "RelDataType", join_type: str = None
```

Comment:
Should the type annotation here be

```suggestion
        cc: ColumnContainer, row_type: "RelDataType", join_type: str | None = None
```

Since `join_type` is an optional argument?

Reply:
Not sure why this change isn't passing CI but it looks like we do the `argument: str = None` for optional strings in several places ex: https://github.com/dask-contrib/dask-sql/blob/5421bbf9b363ab15c657432e0c9d367c6f236df7/dask_sql/context.py#L164 maybe to avoid this issue.

Reply:
Ah apologies I saw this syntax being used in other places but looks like this requires `python>=3.10`; think we could instead do something like

```suggestion
        cc: ColumnContainer, row_type: "RelDataType", join_type: Optional[str] = None
```